src/Styling/SubMenu.css

```diff
@@ -0,0 +1,41 @@
+.Menu_item_links {
+  display: flex;
+  color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  list-style: none;
+  height: 60px;
+  text-decoration: none;
+  font-size: 18px;
+  font-family: 'Poppins';
+}
+
+.Menu_item_links:hover { 
+    background: #252831;
+    border-left: 4px solid #DE5B31 ;
+    cursor: pointer;
+}
+
+.Menu_items {
```

Comment:
This classname is only being used on “item.title” in your SubMenu.JS file. 
1) This seems like a good scenario to use an ID instead of a className
2) Consider changing the name to something more descriptive of its implementation?? (e.g. Item_title)